doc/source/reference/functions/time_classify_day.rst

```diff
@@ -8,11 +8,111 @@
 Summary
 -------
 
+``time_classify_day`` rounds date to a day unit.
+
+That is rounded "2020-01-30 11:50:11.000000" and "2020-01-30 22:50:11.000000" to "2020-01-30 00:00:00.000000".
```

Comment:
"This rounds ..."

Reply:
`"2020-01-30 11:50:11.000000"` -> ``` ``2020-01-30 11:50:11.000000`` ```

Reply:
Thank you for your comment.
I've fixed it.